src/map-model.js

```diff
@@ -46,6 +46,21 @@ MainMap.getTypeFilter = function () {
     return ['==', ['get', filterBy], true];
 }
 
+MainMap.prototype.makeZoomToNationalButton = function () {
+    document.querySelector('.mapboxgl-ctrl-compass').remove();
+    if (document.querySelector('.mapboxgl-ctrl-usa')) {
+        document.querySelector('.mapboxgl-ctrl-usa').remove();
+    }
+    var usaButton = document.createElement('button');
+    usaButton.className = 'mapboxgl-ctrl-icon mapboxgl-ctrl-usa';
+
+
+    usaButton.innerHTML = '<span class="usa-icon"></span>';
+
+    usaButton.addEventListener('click', () => this.resetView());
```

Comment:
Let's update the duration for the resetView method to be 1000 ms (rather than 2000) before merging down. Other than that, looks good and approved in advance.